doc/reference/modules/configuration.xml

```diff
@@ -1081,6 +1081,20 @@ in the parameter binding.</programlisting>
                             </para>
                         </entry>
                     </row>
+                    <row>
+                        <entry>
+                            <literal>sqlite.binaryguid</literal>
```

Comment:
This new setting lacks its declaration in nhibernate-configuration.xsd file. Thus, it cannot be used in configuration files, and can only be used programmatically. Fixing this is just one line to add into nhibernate-configuration.xsd.
Ideally, it should be done on 5.2.x. But I do not think it deserves a release on its own. So I propose to handle it with a PR targeting master, and waiting for 5.3. If we have a 5.2.x to do for some other reasons before, we could then retarget 5.2x. Otherwise just fix it for 5.3. @oskarb